### PR TITLE
feature(misc-apps): Add prom2teams to send alerts to Microsoft Teams

### PR DESCRIPTION
**Pull request by isantospardo**
# Description

Add chart for prom2teams to forward Prometheus Alerts to Microsoft Teams using defined connectors 

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/misc-apps/values.yaml [hunk 1]
```diff
@@ -96,3 +96,22 @@ chartmuseum:
...
+  name: prom2teams
+  destination:
+    # prom2teams.destination.namespace -- Namespace
+    namespace: "infra-prom2teams"
+  # prom2teams.repoURL -- [repo](https://prometheus-msteams.github.io/prometheus-msteams)
+  repoURL: https://prometheus-msteams.github.io/prometheus-msteams
+  # prom2teams.chart -- Chart
+  chart: prom2teams
+  # prom2teams.targetRevision -- [prom2teams Helm chart](https://github.com/idealista/prom2teams/tree/master/helm)
+  targetRevision: "3.2.*"
```
Comment (hairmare) [inline charts/misc-apps/values.yaml:114]: The chart from https://prometheus-msteams.github.io/prometheus-msteams  seems to be for deploying this prom2teams: https://github.com/prometheus-msteams/prometheus-msteams

But version 3.2.3 from https://github.com/idealista/prom2teams/ is seemingly a different implemetation of prom2teams

Which version do we intend to deploy? Can we compare the two versions to make an informed decision?

In any case the PR needs to be updated to only use on of them.

**Review by hairmare — CHANGES_REQUESTED** [review]

File: charts/misc-apps/values.yaml (hunk 1 above)
Comment (isantospardo) [reply charts/misc-apps/values.yaml]: Thanks for noticing, I just updated it.

**Comment by isantospardo** [comment]
Ready for review

File: charts/misc-apps/values.yaml [hunk 2]
```diff
@@ -96,3 +96,22 @@ chartmuseum:
   # chartmuseum.values -- Helm values
   # @default -- [upstream values](https://github.com/chartmuseum/charts/blob/main/src/chartmuseum/values.yaml)
   values: {}
+
+# prom2teams -- [prom2teams](https://github.com/prometheus-msteams/prometheus-msteams) ([example](./example/prom2teams.yaml))
+# @default -- -
+prom2teams:
```
Comment (eyenx) [inline charts/misc-apps/values.yaml:102]: how did we end up with this name? the project's name is prometheus-msteams

Comment (isantospardo) [reply charts/misc-apps/values.yaml:102]: We can always change it, I ended up using this as previously I was using this project instead https://github.com/idealista/prom2teams , but I am changing it then, thanks for the suggestion.

**Review by eyenx — APPROVED** [review]
LGTM

**Comment by eyenx** [comment]
This needs to be fixed asap. there is not TEMPLATE!

**Pull request merged**